libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -205,6 +227,10 @@ void AP_UAVCAN::init(uint8_t driver_index, bool enable_filters)
     AP_BattMonitor_UAVCAN::subscribe_msgs(this);
     AP_Airspeed_UAVCAN::subscribe_msgs(this);
 
+    imu_delta_publisher[driver_index] = new uavcan::Publisher<org::ardupilot::sensor_output::IMUDeltas>(*_node);
```

Comment:
shouldn't creating this object depend on the mask being non-zero?

Reply:
It doesn't for any of the other publishers.